src/vueWrapper.ts

```diff
@@ -223,7 +223,9 @@ export class VueWrapper<T extends ComponentPublicInstance>
   findAll(selector: string): DOMWrapper<Element>[] {
     const results = this.parentElement['__vue_app__']
       ? this.parentElement.querySelectorAll(selector)
-      : this.element.querySelectorAll(selector)
+      : this.element.querySelectorAll
+      ? this.element.querySelectorAll(selector)
+      : ([] as unknown as NodeListOf<Element>)
```

Comment:
nit: is this cast really necessary here? I would have thought that returning a simple array would be enough.

Reply:
I gave it a quick try, and it looks like it's unnecessary. I'll make a PR to remove it.

Reply:
My bad: it _was_ necessary!

Reply:
Yeah, tsd doesn't like it :-P